src/extensions/default/bramble/lib/LinkManagerRemote.js

```diff
@@ -4,7 +4,11 @@
     "use strict";
 
     function handleClick(e) {
-        var url = e.target.getAttribute("href");
+        if(e.target.tag !== "A") {
```

Comment:
If you have something like `<a href="whatever">Go to <code>&lt;div&gt;</code> </a>` and you click on the div part, the event is triggered first for the child and only then the parent (&lt;a&gt; tag). For the child, it will get an empty url and open it in the new tab.